pkg/obfuscate/sql_tokenizer.go

```diff
@@ -387,6 +387,9 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 			if kind == DollarQuotedFunc {
 				// this is considered an embedded query, we should try and
 				// obfuscate it
+				// Trim off the $func$ tag
+				tok = tok[6:]
```

Comment:
This is a bit safer and more explicit: (and removes the need for the comment explaining the code above)

```
tok = tok[len("$func$"):]
tok = tok[:len(tok)-len("$func$")]
```